app/definitions/ISubscription.ts

```diff
@@ -73,8 +73,8 @@ export interface ISubscription {
 	lastThreadSync?: Date;
 	jitsiTimeout?: number;
 	autoTranslate?: boolean;
-	autoTranslateLanguage: string;
-	lastMessage?: ILastMessage;
+	autoTranslateLanguage?: string;
+	lastMessage?: ILastMessage; // TODO: we need to use IMessage here
```

Comment:
I changed to `IMessage` and runned `yarn lint` no errors was found. Did you forgot to remove this `TODO`?